base/string_utils.cpp

```diff
@@ -326,4 +326,29 @@ bool AlmostEqual(string const & str1, string const & str2, size_t mismatchedCoun
   return false;
 }
 
+void Split(string const & s, char delimiter, vector<string> & target)
```

Comment:
When you need quite basic sorting you use std::sort, not writing your own implementation, right?